README.md

```diff
@@ -1,44 +1,72 @@
 **Português (BR)** | [English (US)](.github/README-en-US.md)
 
 <p align="center">
-  <img alt="Querido Diário" src="./.github/images/querido-diario-logo.png" width="200">
+  <a href="https://queridodiario.ok.org.br/sobre" target="_blank"> <img alt="Querido Diário" src="./.github/images/querido-diario-logo.png" width="200">
+  </a>
 </p>
 
 # Documentação e Comunidade
 
-Documentação mais geral do Querido Diário e espaço dedicado a [discussões](https://github.com/okfn-brasil/querido-diario-comunidade/discussions) e à
-comunidade do projeto.
+Dentro do [ecossistema do Querido Diário](.github/CONTRIBUTING.md#ecossistema), este repositório é o responsável pela **documentação geral** e pelas **[discussões](https://github.com/okfn-brasil/querido-diario-comunidade/discussions)** transversais aos vários repositórios do projeto e sua comunidade.
+
+Conheça mais sobre as [tecnologias](https://queridodiario.ok.org.br/tecnologia) e a [história](https://queridodiario.ok.org.br/sobre) do projeto no [site do Querido Diário](https://queridodiario.ok.org.br)
 
-*A documentação está em progresso.* :building_construction:
+# Sumário
+- [Como contribuir](#como-contribuir)
+- [Suporte](#suporte)
+- [Agradecimentos](#agradecimentos)
+- [Open Knowledge Brasil](#open-knowledge-brasil)
+- [Licença](#licença)
 
-# Contribuição
+
+# Como contribuir
+<p>  
+  <a href="https://www.catarse.me/projects/120548/" target="_blank"> 
+    <img alt="catarse" src="https://img.shields.io/badge/Doe-Catarse-green" width="100">
+  </a>
+</p> 
```

Comment:
Dei uma lida nesse [shields.io](https://shields.io/) e entendi melhor como se usa as badges.

As palavras que aparecerão seguem o formato `https://img.shields.io/badge/<LABEL>-<MESSAGE>-<COLOR>`, onde `LABEL` aparece na esquerda e `MESSAGE` na direita. O `COLOR` é a cor da `MESSAGE`.

Também temos os estilos, o que estamos usando é o `for-the-badge` e podemos manter ele como principal. O `flat-square` também parece legal mas é bem mais discreto e talvez possamos usar quando for o caso.

![image](https://user-images.githubusercontent.com/26327506/192828957-c3e791c2-906d-4386-b9ba-c73f240a3135.png)

Também dá pra configurar outras coisas como a logo que aparece, cor da label, etc. Isso pode ser feito na querystring do shield:

Parâmetro | Descrição
-- | --
?label=healthinesses | Override the default left-hand-side text (URL-Encoding needed for spaces or special characters!)
?logo=appveyor | Insert one of the named logos from (bitcoin, dependabot, gitlab, npm, paypal, serverfault, stackexchange, superuser, telegram, travis) or simple-icons. Simple-icons are referenced using icon slugs which can be found on the simple-icons site or in the slugs.md file in the simple-icons repository.
?logo=data:image/png;base64,… | Insert custom logo image (≥ 14px high). There is a limit on the total size of request headers we can accept (8192 bytes). From a practical perspective, this means the base64-encoded image text is limited to somewhere slightly under 8192 bytes depending on the rest of the request header.
?logoColor=violet | Set the color of the logo (hex, rgb, rgba, hsl, hsla and css named colors supported). Supported for named logos but not for custom logos.
?logoWidth=40 | Set the horizontal space to give to the logo
?link=http://left&link=http://right | Specify what clicking on the left/right of a badge should do. Note that this only works when integrating your badge in an<object> HTML tag, but not an<img> tag or a markup language.
?labelColor=abcdef | Set background of the left part (hex, rgb, rgba, hsl, hsla and css named colors supported). The legacy name "colorA" is also supported.
?color=fedcba | Set background of the right part (hex, rgb, rgba, hsl, hsla and css named colors supported). The legacy name "colorB" is also supported.
?cacheSeconds=3600 | Set the HTTP cache lifetime (rules are applied to infer a default value on a per-badge basis, any values specified below the default will be ignored). The legacy name "maxAge" is also supported.

Tentei colocar uma logo customizada do Catarse e dá pra fazer. Só tem que encontrar uma imagem pequena e de fundo transparente (PNG ou ICO), [converter pra base64](https://onlineimagetools.com/convert-image-to-base64) e colar o resultado no campo `logo`.

Exemplo de como usar uma logo customizada e com `label` + `message`:

![image](https://user-images.githubusercontent.com/26327506/192830829-047b7bbf-3d9c-47f3-a6ce-74b6a1071c68.png)

```suggestion
<p>  
  <a href="https://www.catarse.me/queridodiario-okbr" target="_blank"> 
    <img alt="catarse" src="https://img.shields.io/badge/Catarse-Apoie o projeto-orange?style=for-the-badge&logo=data:image/ico;base64,AAABAAEAEBIAAAEAIAAXAgAAFgAAAIlQTkcNChoKAAAADUlIRFIAAAAQAAAAEggGAAAAUjteagAAAd5JREFUOI2NlLFrU1EUh7/z7mtIQiWh1ae0SKJBxBaxzyUgBUsULOjQoeCm6V9g1cVB8AUXwSFxcRBB+gYRB3Vw0Mk4OAhCsuiiKVFEsBFqwYJJTa5Dk9fXlzTNb7r33PP77jmcyxWtNSJCUBXbmQIKwGngK5BNlZwigNbayzO6nFt60TYDJIA3FdvJBpP6ARI9YoWK7SQHBZR7xGKAMyhgAaj2iF9ePpmL7wpIlZwyYAPF4Jk5+nG+s5adpuBXxXYeAdnOPnL0CWrPt6vWg/VCvxb81SyIanzxblV/ARb7thCUGv5+f9Ncx4iuQHtKAwMSRTcPMLT/QydU3gaw3PSc5abj3dYthcbeeQBRPPUAlpueAZ6vXHr/ezeAqDoS1gwd2Tjvr6AwagoXnp2a3slcuxa5A2yax5uIYtIPODEWEsZDstTL/OtG+CJNrpsHmoSS/5CwBrgHYLZzqvUWyX2mHH74euLTpKo9njBqL/UPNUODWf40z9DycgGc2JVGzl/BUrXeoqHRq0SPpYzV20BJIjoPnCOqTYY17G0BHIovbuQ6JO8lWm56an7EPDtiyl1L1smoZQ4aa8FuyrEMtv8/6HrKN19NzwG3gORx42d+1vy8BsSBYizDW9j+ofwHqM6MbbvUDjEAAAAASUVORK5CYII=" width="100">
  </a>
</p> 
```